### ISSUE TITLE
Very large loss in Convolutional Neural Network

### ISSUE THREAD

**Issue by NithinShrivatsav** [jeromeyoon/Tensorflow-siamese#4]
I modified your code to incorporate a convolutional neural network, but the loss is very high and accuracy is very low? What could be the reason for this?
